SmartDeviceLink/SDLLockScreenManager.m

```diff
@@ -147,6 +169,41 @@ - (void)sdl_checkLockScreen {
     }
 }
 
+- (void)sdl_updateLockScreenDismissable {
+    if (self.lastDriverDistractionNotification == nil || self.lastDriverDistractionNotification.lockScreenDismissalEnabled == nil ||
+        ![self.lastDriverDistractionNotification.lockScreenDismissalEnabled boolValue]) {
```

Comment:
```suggestion
        !self.lastDriverDistractionNotification.lockScreenDismissalEnabled.boolValue) {
```

These conditionals should all be on one line or separated onto three lines as well.